doc/user-guide/jupyter/itk_plotting.rst

```diff
@@ -26,7 +26,8 @@ Special thanks to `@thewtex`_ for the `itkwidgets`_ library.
 Installation
 ++++++++++++
 To use `PlotterITK` you'll need to install ``itkwidgets>=0.25.2``.
-Follow the installation steps `here <https://github.com/InsightSoftwareConsortium/itkwidgets#installation>`_.
+Follow the installation steps `here <https://itkwidgets.readthedocs.io/en/latest/quick_start_guide.html>`_.
```

Comment:
This also seems to break linkcheck due to the anchor. It's fine to replace it with the doc link though.

Reply:
do you mean this link is okay or do you refer to another link?

Reply:
I was just pointing out that it was again the anchor that broke this. But replacing with the new link as you did is fine. So nothing to do here, just noting the same issue as fixed elsewhere with the `user-content` prefix for the fragment.